setup-homebrew/main.mjs

```diff
@@ -0,0 +1,13 @@
+import {exec} from "@actions/exec"
+import core from "@actions/core"
+
+try {
+  await exec("bash", [
+    new URL("./main.sh", import.meta.url).pathname,
+    core.getInput("test-bot"),
+    core.getInput("debug"),
+    core.getInput("token")
+  ])
+} catch (error) {
+  core.setFailed(error.message)
+}
```

Comment:
I don't really understand why this is needed, can you elaborate a bit?

Reply:
Unfortunantely, GitHub does not support `post` actions in shell so a JS wrapped is needed.